src/main/java/com/hotels/mutantswarm/report/ReportGenerator.java

```diff
@@ -96,7 +96,11 @@ private File resolveReportFolder() {
   }
 
   private File resolveReportFile(File reportFolder) {
-    String reportFileName = String.format(reportFileNameformat, "mutation_report");
+    String className = results.getSuiteName();
+    if (className.contains("class ")) {
```

Comment:
You don't need this `if` statement, if the string isn't there then `replace` just returns the original string.